auslib/web/public/client.py

```diff
@@ -29,6 +29,33 @@ def getHeaderArchitecture(buildTarget, ua):
         return 'Intel'
 
 
+def getSystemCapabilities(systemCapabilities):
```

Comment:
I don't think that's a great name, because it does more than that (it also returns memory, and probably other values in the future). I'm open to other suggestions though - now that I re-read this, a function called "getSystemCapabilities" that takes an argument of systemCapabilities is very weird...